.gitattributes

```diff
@@ -0,0 +1 @@
+*.sh        text eol=lf
```

Comment:
As this is not directly related to the PR at hand, I'd rather have this in a separate PR to keep the git history cleaner.

Reply:
ok. I'll create a separate branch/PR. Let me know if I need to do anything else to this branch.

Reply:
This is otherwise good, please remove that commit from this PR and I'll merge it, thanks!

Reply:
I think it's removed.

Reply:
Thanks, I'm currently tight on time but I'll try to check out your other PRs by the end of the next week.